.zenodo.json

```diff
@@ -589,24 +634,22 @@
         {
             "name": "Yash Kothari"
         },
+        {
+            "name": "Yash Malik",
+            "orcid": "0009-0005-7611-034X"
+        },
         {
             "name": "Yukie Nomiya",
             "orcid": "0000-0002-9572-6300",
             "affiliation": "Sapienza University of Rome, Italy"
         },
         {
-            "name": "mridulpandey"
+            "name": "Zach Burnett"
```

Comment:
@zacharyburnett, do you have a ORCID and an affiliation?

Reply:
I don't have an ORCID, and you can put my affiliation as STScI. That name is fine, thanks!